extensions/frontend-tiles/src/BatchedSpatialTileTreeRefs.ts

```diff
@@ -311,8 +318,26 @@ async function fetchTilesetSpec(iModel: IModelConnection, computeBaseUrl: Comput
   }
 }
 
+class EmptySpatialTileTreeReferences implements SpatialTileTreeReferences {
+  public update(): void {}
+
+  public setDeactivated(_modelIds: string | string[] | undefined, _deactivated: boolean | undefined, _refs: "all" | "animated" | "primary" | "section" | number[]): void {}
```

Comment:
Just FYI, no action needed - you can omit the args if you don't use them, e.g.,
`public setDeactivated(): void { }`